section2/reflection.md

```diff
@@ -2,14 +2,30 @@
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
+- I found myself frustrated when I could not get node to run correctly.  I reached out to a friend who is a current Turing student and we talked through it together.  After our discussion, I realized it was an easy solution.  I do not mind having to search for answers or problem solve on my own.  Problem solving simply supports learning, I will remember the process a lot easier, too!***
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+- A conditional statement is involved in decision making - if/then/else will execute if the condition is met.  I would compare a conditional statement to backpacking: if a hike is under 10 miles, you will not need to set up camp.  If a hike is over 10 miles, you will need to set up camp. A conditional is probably used in an application like DuoLingo.  You have to score a certain amount to continue to the next lesson.***
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+- To add multpiple conditions to an 'if' statement you could use logical operators.  There is Logical AND &&  which will read left to right so if it reads True True, the output will also be true.  Both conditions must be true for it to read true.  Any other combination of True and False will result as false.  We can also use an OR operator ||.  The OR operator will run true if one of the operands runs true.  Additionally, if you run the first statement and the results are false, it would move to the NEXT if statement.  If the second statement runs false, it would move to the NEXT statement (if else) and etc.  It will run through until it logs a statement as true.
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+- Being able to run NODE in the terminal was extremely helpful while adjusting the code in Atom, Google Chrome Developer Tools is another excellent resource, you are able to run code in your browser.  You could also have a peer review it!
+
+7. What questions do you still have about `if` statements and/or functions?
+
+- When dealing with multiple variables in conditional statements - especially when one value is true or false, are logical operators the best option?
+- When do you determine using IF vs ELSE IF
```

Comment:
`if` is the condition in the statement, from there you will use `else if`'s for each additional scenario.